The declaration of `operator<=>` in base::flat_tree is removed in the
previous patch, so do not use `base::flat_tree::operator<=>` in the
class which uses `base::flat_tree`.

--- a/components/viz/common/view_transition_element_resource_id.h
+++ b/components/viz/common/view_transition_element_resource_id.h
@@ -33,8 +33,11 @@
   VIZ_COMMON_EXPORT friend bool operator==(
       const ViewTransitionElementResourceId&,
       const ViewTransitionElementResourceId&);
-  friend auto operator<=>(const ViewTransitionElementResourceId&,
-                          const ViewTransitionElementResourceId&) = default;
+  friend bool operator<(const ViewTransitionElementResourceId& lhs,
+                        const ViewTransitionElementResourceId& rhs) {
+    return std::tie(lhs.transition_token_, lhs.local_id_) < 
+            std::tie(rhs.transition_token_, rhs.local_id_);
+  }
 
   bool IsValid() const;
   std::string ToString() const;
